scripts/start.py

```diff
@@ -139,6 +143,13 @@ def main(args=None):
             '--config', 'webpack.dev.config.ts', '--watch']))
         # Give webpack few seconds to do the initial compilation.
         time.sleep(10)
+        # Redis-cli is only required in a development environment.
+        python_utils.PRINT('Starting Redis development server')
+        # Not appended to background_processes since this server is not a python
+        # backend process. It does not hang the terminal but returns
+        # automatically and runs the service in the system background.
+        subprocess.Popen(
+            'redis-server %s' % (common.REDIS_CONF_PATH), shell=True)
```

Comment:
Is this server killed when `redis-cli shutdown` is called?

Reply:
Yes. This is the only way, redis-server does not have a shutdown command for some reason. The only way to shut it down is through redis-cli.